src/main/java/com/scrive/pdftools/tools/flattener/FlattenerHandler.java

```diff
@@ -0,0 +1,17 @@
+package com.scrive.pdftools.tools.flattener;
+
+import org.json.JSONObject;
+
+import java.io.ByteArrayOutputStream;
+
+public class FlattenerHandler {
+
+    public static ByteArrayOutputStream execute(JSONObject flatSpecJSON) throws Exception {
+        FlattenerSpec flatSpec = FlattenerSpec.fromJson(flatSpecJSON);
+        if (flatSpec == null) {
+            throw new Exception("Failed to parse spec from json");
+        }
+        ByteArrayOutputStream res = LambdaFlattener.execute(flatSpec);
+        return res;
```

Comment:
I guess we can avoid the intermediate value and directly return
```suggestion
        return LambdaFlattener.execute(flatSpec);
```

Reply:
Yes.